configs/batch-llm/aws.yaml

```diff
@@ -1,84 +1,4 @@
 head_node_type:
-  name: head-node
-  instance_type: m5.2xlarge
-  resources:
-    cpu: 0
-worker_node_types:
-  - name: worker-g5-xlarge-nvidia-a10-1
-    instance_type: g5.xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-2xlarge-nvidia-a10-1
-    instance_type: g5.2xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-4xlarge-nvidia-a10-1
-    instance_type: g5.4xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-8xlarge-nvidia-a10-1
-    instance_type: g5.8xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-12xlarge-nvidia-a10-4
-    instance_type: g5.12xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 1
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-16xlarge-nvidia-a10-1
-    instance_type: g5.16xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-24xlarge-nvidia-a10-4
-    instance_type: g5.24xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 1
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-48xlarge-nvidia-a10-8
-    instance_type: g5.48xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 1
-    use_spot: true
-    fallback_to_ondemand: true
-aws:
-  TagSpecifications:
-    - ResourceType: instance
-      Tags:
-        - Key: as-feature-multi-zone
-          Value: "true"
+  name: head
+    # TODO(ricky): We need head node to have CUDA due to eager import from rayllm_batch now.
+  instance_type: g5.xlarge
```

Comment:
we generally don't want to encourage the pattern of using GPU head nodes --> since it leads to people running code on head nodes that take down the workload due to oom (antipattern)

can you use a CPU head node instead with scheduling disabled (see basic serverless config that existing templates use) and wrap your code in an actor or something that runs on workers?

Reply:
This makes sense - instead of temp fixing this by wrapping in actor, I think we will fix this by addressing the root cause soon, that is to make this code runnable on CPU itself (which it should be, it's just we are not lazy importing vllm as of now).